backend/serverless/lambdas/users/updateThumbnails/data/repositories/imagesTransformation/repository.js

```diff
@@ -1,8 +1,12 @@
 const axios = require('axios');
 
 module.exports.init = function init() {
-  async function cropImage(imageUrl) {
-    const baseUrl = 'https://res.cloudinary.com/demo/image/fetch/c_fill,g_face,h_100,w_100/r_max/f_auto';
+  async function cropImage({
+    imageUrl,
+    width = 100,
+    height = 100,
+  }) {
+    const baseUrl = `https://res.cloudinary.com/demo/image/fetch/c_fill,g_face,h_${height},w_${width}/r_max/f_auto`;
```

Comment:
I think we could extract the cloudinary link part to a constant and/or a configuration value